msticpy/sectools/vtlookupv3.py

```diff
@@ -15,7 +15,7 @@ class MsticpyVTNoDataError(Exception):
 
 
 class MsticpyVTGraphSaveGraphError(Exception):
-    """Could not save VT Garph."""
```

Comment:
I did some updates to this to get rid of some linting/black warnings - you might want to check that I fixed this typo when merging